### ISSUE TITLE
Generating FHIR Profiles based on FHIR Connect Mappings

### ISSUE THREAD

**Issue by gasperr** [medblocks/openFHIR#72]
An idea that came from _the community_ and I think it's a nice one - it should be possible to create FHIR profiles based on fhir connect mappings.

**Comment by yampeku** [comment]
Yeah I would assume is much safer to be the models or profiles the ones that autogenerate the mappings and not the other way around
